to-do-list/test.js

```diff
@@ -6,10 +6,10 @@
 
 Revisar capitulo 3 - Ceviche.js
 */
-let itemLink = document.getElementById('addItem');
-let newItem = document.getElementById('item');
+const itemLink = document.querySelector('#addItem');
+let newItem = document.querySelector('#item');
```

Comment:
Given you are not replacing `newItem`'s value, you can define it with `const` too.